backend/src/main/java/net/finman/dao/InvoiceDaoImpl.java

```diff
@@ -92,37 +99,37 @@ public void addInvoiceItems(UUID serialNumber, String seller, List<Item> items)
             SqlParameterSource invoiceParams = new MapSqlParameterSource()
                     .addValue("source", source)
                     .addValue("email", source);
-            
+
             InvoiceMapper mapper = new InvoiceMapper();
             List<Invoice> invoices = template.query(GET_INVOICES, invoiceParams, mapper);
 
-            // add the items to the invoices
-            for (Invoice invoice : invoices){
-                invoice.setItems(getInvoiceItems(invoice.getSerialNumber(), invoice.getSeller()));
-            }
-            
+            // //add the items to the invoices
+            // for (Invoice invoice : invoices){
+            //     invoice.setInvoiceItems(getInvoiceItems(invoice.getSerialNumber(), invoice.getSeller()));
+            // }
+
             if (invoices.size() == 0)
                 throw new ResourceNotFoundException("You don't have any invoices!", "");
 
             return invoices;
         } catch(DataAccessException e) {
-           throw new ResourceNotFoundException("Error communcating with database!", e.getMessage());
+            throw new ResourceNotFoundException("Error communcating with database!", e.getMessage());
         }
     }
 
     @Override
-    public List<Item> getInvoiceItems(UUID invoice, String seller) throws ResourceNotFoundException {
+    public List<InvoiceItem> getInvoiceItems(UUID serial_no, String seller) throws ResourceNotFoundException {
         try {
             SqlParameterSource invoiceItemParams = new MapSqlParameterSource()
-                    .addValue("invoice", invoice)
+                    .addValue("serial_no", serial_no)
                     .addValue("seller", seller);
-            
+
             ItemMapper mapper = new ItemMapper();
-            List<Item> items = template.query(GET_INVOICE_ITEMS, invoiceItemParams, mapper);
+            List<InvoiceItem> items = template.query(GET_INVOICE_ITEMS, invoiceItemParams, mapper);
 
             return items;
         } catch(DataAccessException e) {
-           throw new ResourceNotFoundException("Cannot retrieve invoice items!", e.getMessage());
+            throw new ResourceNotFoundException("Cannot retrieve invoice items!", e.getMessage());
         }
     }
```

Comment:
We don't need this function `getInvoiceItems(...)` anymore since invoiceItems are retrieved through an Invoice directly